.github/workflows/windows_test_comment_trigger.yml

```diff
@@ -0,0 +1,87 @@
+            repo,
+            comment_id: context.payload.comment.id,
+            content: "rocket",
+          });
+    - name: get_pr_sha
+      id: sha
+      uses: actions/github-script@v4
+      with:
+        result-encoding: string
+        script: |
+          const { owner, repo, number } = context.issue;
+          const pr = await github.pulls.get({
+            owner,
+            repo,
+            pull_number: number,
+          });
+          return pr.data.head.sha
+    - name: update_pr_status_pending
+      run: |
+          gh api --method POST -H "Accept: application/vnd.github+json" /repos/$OWNER/$REPO/statuses/$SHA -f state='pending' -f target_url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}' -f description='The build started2!' -f context='Windows pre-commit'
+          echo "REF == = = = = = = ${{ steps.sha.outputs.result }}"
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        OWNER: ${{ github.repository_owner }}
+        REPO: ${{ github.event.repository.name }}
+        SHA: ${{ steps.sha.outputs.result }}
+    outputs:
+      PR_SHA: ${{ steps.sha.outputs.result }}
+
+
+  windows_default:
+    name: Windows
+    needs: [windows_test_preparation]
+    if: github.repository == 'intel/llvm'
+    uses: ./.github/workflows/sycl_windows_build_and_test.yml
+    with:
+      build_ref: ${{ needs.windows_test_preparation.outputs.PR_SHA }}
+
+  windows_test_completion:
+    runs-on: ubuntu-latest
+    needs: [windows_default]
+    steps:
+    - name: update_pr_status_success
+      if: always() && success()
+      run: |
+          gh api --method POST -H "Accept: application/vnd.github+json" /repos/$OWNER/$REPO/statuses/$SHA -f state='success' -f target_url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}' -f description='The build started2!' -f context='Windows pre-commit'
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        OWNER: ${{ github.repository_owner }}
+        REPO: ${{ github.event.repository.name }}
+        SHA: ${{ steps.sha.outputs.result }}
+    - name: update_pr_status_failure
+      if: failure()
+      run: |
+          gh api --method POST -H "Accept: application/vnd.github+json" /repos/$OWNER/$REPO/statuses/$SHA -f state='failure' -f target_url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}' -f description='The build started2!' -f context='Windows pre-commit'
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        OWNER: ${{ github.repository_owner }}
+        REPO: ${{ github.event.repository.name }}
+        SHA: ${{ steps.sha.outputs.result }}
+
+  # windows_default:
+  #   runs-on: ubuntu-latest
+  #   needs: [windows_test_preparation]
+  #   steps:
+  #   - name: print_ref
+  #     run: |
+  #       echo "REF == = = = = = = ${{github.ref}}"
+  #       echo "REF == = = = = = = ${{ needs.windows_test_preparation.outputs.PR_SHA }}"
```

Comment:
@romanovvlad, can we remove this commented out part of the script?

Reply:
Yes. It was done here: https://github.com/intel/llvm/commit/017bcdcfbaf3c400ae67e7cf4ca135e623c5ecc6